frontend/src/components/contactsList.tsx

```diff
@@ -50,39 +79,129 @@ export const ContactsList = (): JSX.Element => {
+                <FontAwesomeIcon icon={faUser} className='viewIcons' style={{ color: '#c8cbd0', marginRight: '8px' }} />
+                Contact
+              </button>
+            </div>
+
+            <div className="recentView">
+              <button className={`viewButtons ${buttonStates[1] ? 'clicked' : ''}`} onClick={() => { handleButtonClick(1) }}>
+                <FontAwesomeIcon icon={faClockRotateLeft} className='viewIcons' style={{ color: '#c8cbd0', marginRight: '5px' }} />
+                Recent
+              </button>
+            </div>
+
+            <div className="favoriteView">
+              <button className={`viewButtons ${buttonStates[2] ? 'clicked' : ''}`} onClick={() => { handleButtonClick(2) }}>
+                <FontAwesomeIcon icon={faStar} className='viewIcons' style={{ color: '#c8cbd0', marginRight: '4px' }} />
+                Favorite
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <div className="rightBar">
+          <div className="rightBarHeader">
+            <p className='pName'>Name</p>
+            <p className='pEmail'>Email</p>
+            <p>Phone number</p>
+            <p>Relationship</p>
+            <div className="navigationButtons">
+              <button className='navButtons'>
+                <FontAwesomeIcon icon={faCloudArrowDown} style={{ color: '#8C8787' }} />
+              </button>
+              <button className='navButtons'>
+                <FontAwesomeIcon icon={faShareFromSquare} style={{ color: '#8C8787' }} />
+              </button>
+              <button className='navButtons'>
+                <FontAwesomeIcon icon={faArrowUpWideShort} style={{ color: '#8C8787' }} />
+              </button>
+            </div>
+          </div>
+          <hr className="mainHorizontalLine"></hr>
+
+          <div className="listOfContacts">
+            {contacts?.map((contact) => (
+              <div key={contact.id} className="contactItem">
+                <img className="contactIcon" src={contactIcon} alt="contactIcon" />
+                  <div className="contactInfo">
+                    <div className='dName'> {contact.firstName} {contact.lastName}</div>
+                    <div className='dEmail'> {contact.email}</div>
+                    <div className='dPhone'> {contact.phoneNumber}</div>
+                    {contact.relationships !== null && (
+                      <div className="dRelationship">
+                        {contact.relationships?.map((relationship) => (
+                          <div key={relationship.id}> {relationship.relationshipType}</div>
+                        ))}
+                      </div>
+                    )}
+                    <div className="favoriteIcon" onClick={() => { handleToggleFavorite(contact.id); }}>
+                      {contact.isFavorite !== null && contact.isFavorite !== undefined && contact.isFavorite
```

Comment:
Kind of confused here, why would isFavorite be null or undefined here? I think it goes back to the question: should contacts have a default state of being favorited or not-favorited.

Lets say the default state is false, then we can write the code here a lil cleaner with a simple `contacts.isFavorite ? yellow star : white star`

Reply:
Yeah I see what you mean I will change it now since I have set the default state to false. I will update this in my PR for favoritedContacts to keep it more organized.